config/default.js

```diff
@@ -5,6 +5,8 @@
  *
  * Documentation: https://github.com/lorenwest/node-config/wiki/Configuration-Files
  */
+
+var NODE_ENV = process.env.NODE_ENV
```

Comment:
Can we please use `const` here instead of `var` since the value will not be changed?

Reply:
Yes. Will carry out the changes. Thanks for the help!